bin/submit_gpu_job

```diff
@@ -13,38 +16,51 @@ TWO_DAYS = 2 * 24 * 60
 def write_slurm_template(script, out_path, arguments,
                          n_threads, gpu_type, n_gpus,
                          mem_limit, time_limit, qos,
-                         mail_address, exclude_nodes):
-    slurm_template = ("#!/bin/bash\n"
-                      "#SBATCH -A kreshuk\n"
-                      "#SBATCH -N 1\n"
-                      "#SBATCH -c %s\n"
-                      "#SBATCH --mem %s\n"
-                      "#SBATCH -t %i\n"
-                      "#SBATCH --qos=%s\n"
-                      "#SBATCH -p gpu\n"
-                      "#SBATCH -C gpu=%s\n"
-                      "#SBATCH --gres=gpu:%i\n") % (n_threads,
-                                                    mem_limit, time_limit,
-                                                    qos, gpu_type, n_gpus)
+                         exclude_nodes, mail_address, array):
+    user = pwd.getpwuid(os.getuid()).pw_name
+    slurm_out = Path(f"/scratch/{user}/slurm_out")
+    slurm_out.mkdir(parents=True, exist_ok=True)
+    slurm_template = (
+        f"#!/bin/bash\n"
+        f"#SBATCH -A kreshuk\n"
+        f"#SBATCH -N 1\n"
+        f"#SBATCH -o {slurm_out / '%N.%j.out'}\n"
+        f"#SBATCH -e {slurm_out / '%N.%j.err'}\n"
```

Comment:
Redirecting the slurm output without telling the user is not a good practice. We should either make this optional (and default to False) or at least print where the slurm output can be found when running this.